tests/python_package_test/test_dask.py

```diff
@@ -1255,3 +1255,59 @@ def test_sklearn_integration(estimator, check, client):
 def test_parameters_default_constructible(estimator):
     name, Estimator = estimator.__class__.__name__, estimator.__class__
     sklearn_checks.check_parameters_default_constructible(name, Estimator)
+
+
+@pytest.mark.parametrize('task', ['binary_classification', 'multi-class_classification', 'regression'])
+@pytest.mark.parametrize('output', data_output)
+def test_predict_with_raw_score(task, output, client):
+    if task.endswith('classification'):
+        objective = 'classification'
+        model_factory = lgb.DaskLGBMClassifier
+    elif task == 'regression':
+        objective = 'regression'
+        model_factory = lgb.DaskLGBMRegressor
+
+    if task.startswith('multi'):
```

Comment:
```suggestion
    if task == 'multi-class_classification':
```

There is only one task that this matters for, so I'd rather be explicit and check that it is multi-class classification. That reduces indirection and makes the test code easier to follow.